### PR TITLE
write RSpec for order

### PR DESCRIPTION
**Pull request by namtm-1272**
## Related Tickets
- [#53317](https://edu-redmine.sun-asterisk.vn/issues/53317)

## WHAT (optional)
- write test rspec for order model 

## HOW
- I create order_spec.rb in order to test all lines in order model

## WHY (optional)
- Cần test trước để tránh lỗi

## Evidence (Screenshot or Video)
Screenshot from 2022-09-14 10-57-27

### CODE REVIEW

**Comment by namtm-1272** [comment]
ready

File: spec/models/order_spec.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      expect(Order.lastest_order.pluck(:id)).to eq([order3.id, order2.id, order1.id])
+    end
+  end
+
+  describe "#handle_order" do
+    let!(:order5){FactoryBot.create :order, status: 0}
+    let!(:order_detail5){FactoryBot.create :order_detail, order_id: order5.id}
+    context "when status is not complete" do
+      order_params = {"status" => 1}
+      it "some thing" do
```
Comment (huongnt-2545) [inline spec/models/order_spec.rb]: viết rõ tên test case em nhé
fix tương tự bên dưới

**Comment by namtm-1272** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**